core/crypto.mk

```diff
@@ -226,6 +228,7 @@ _CFG_CORE_LTC_HASH := $(call ltc-one-enabled, MD5 SHA1 SHA224 SHA256 SHA384 \
 _CFG_CORE_LTC_MAC := $(call ltc-one-enabled, HMAC CMAC CBC_MAC)
 _CFG_CORE_LTC_CBC := $(call ltc-one-enabled, CBC CBC_MAC)
 _CFG_CORE_LTC_ASN1 := $(call ltc-one-enabled, RSA DSA ECC)
+_CFG_CORE_LTC_EC25519 := $(call ltc-one-enabled, X25519 ED25519)
```

Comment:
Do we really want `ED25519` here? It doesn't seem that `CFG_CRYPTO_ED25519` (and therefore `_CFG_CORE_LTC_ED25519`) is supported in this PR?

Reply:
not really, this was just a placeholder; `ED25519` will be a different PR, will remove from here